From: John Pittman <jpittman@redhat.com>
Date: Mon, 1 Aug 2016 16:35:53 -0400
Subject: Include: blkdev: Removed duplicate 'struct request;' declaration.
References: bnc#1023798,FATE#321463
Patch-Mainline: v4.8-rc1
Git-commit: 6d25ec147e3a71858bed5439c92accd7f739a0a3

In include/linux/blkdev.h duplicate declarations of the request
struct exist.  Cleaned up by removing the second, unneeded
declaration.

Signed-off-by: John Pittman <jpittman@redhat.com>
Signed-off-by: Jens Axboe <axboe@fb.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 include/linux/blkdev.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/linux/blkdev.h b/include/linux/blkdev.h
index 8933c11..be09bdb 100644
--- a/include/linux/blkdev.h
+++ b/include/linux/blkdev.h
@@ -47,7 +47,6 @@ struct pr_ops;
  */
 #define BLKCG_MAX_POLS		2
 
-struct request;
 typedef void (rq_end_io_fn)(struct request *, int);
 
 #define BLK_RL_SYNCFULL		(1U << 0)
-- 
1.8.5.6

